src/PersonObjects/formulas/skill.ts

```diff
@@ -1,37 +1,33 @@
+import clampNumber from "../../utils/helpers/clampNumbers"
```

Comment:
Optional: I have a slight preference for `util/helper/clampNumber` (singular) over plural, and I think that fits the style of the existing code better?